<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_UserName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="currentText">
    <string extracomment="账号"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_PassWord">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Login">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>登   陆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Register">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>213</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Type">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>182</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
